translations/admin.en.yaml

```
@@ -432,6 +432,9 @@ system_columns: 'System columns'
 columns: Columns
 children_grid: 'Children Grid'
 only_children: 'just direct children'
+only_object: 'just objects'
+only_variant: 'just variants object'
+select_objects_type: "Object's type to show"
```

```suggestion
select_objects_type: "Type to show"
```
being already in the context of objects, likely no need to specify it